app/assets/stylesheets/components/_firm.scss

```diff
@@ -0,0 +1,44 @@
+.firm {
```

Comment:
This design is based on a mockup we've been recently supplied specifically for this page. We don't have any mockups for search and wasn't in the scope of the piece of work, so I'm inclined to keep the styles separate until we have direction that the firm profile page and the results page should be more inline. I do agree that they could be more inline however.